### DISCUSSION TITLE
swissup gdpr module not working after upgrade

### DISCUSSION THREAD

**Discussion by arunguru77** [breezefront/community#265] [1. Help]
Hi Support,
we are using breeze theme, but after upgrde the swissup gdpr module not working given 2 issues.
1.Uncaught TypeError: updateGoogleConsent is not a function
2.js error "Uncaught DOMException: Failed to execute 'querySelectorAll' on 'Element': '> .field:not(.captcha):not(.g-recaptcha):not(.field-recaptcha)' is not a valid selector." if we disable  Integrate with Google Consent Mode
https://artuk.wcltest.com/
please help to resolve this issue
thanks in advance

**Comment by vovayatsyuk** [comment]
1. Looks like js are not updated. Try to redeploy static content.
2. You need to update breeze module to the latest version. 

Your site asks for a password, so I cannot see it.
